src/test/basics/RangeSet_test.cpp

```diff
@@ -26,69 +26,132 @@ namespace ripple {
 class RangeSet_test : public beast::unit_test::suite
 {
 public:
-    RangeSet createPredefinedSet ()
+    void
+    testSetAndHas()
```

Comment:
Thanks @mellery451, your suggestion uncovered a bug.  I'm actually now more tempted to look at [Boost ICL](http://www.boost.org/doc/libs/1_64_0/libs/icl/doc/html/index.html) after all to avoid these type of errors when rolling our own.